From 16d346dbe0e8cf73dda78d5165bfc3491695e3b1 Mon Sep 17 00:00:00 2001
From: Atri Bhattacharya <A.Bhattacharya@uliege.be>
Date: Thu, 20 Feb 2020 00:12:08 +0100
Subject: [PATCH] Py3: use a string as the 1st argument to exec [skip ci]

---
 lib/meta.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/meta.py b/lib/meta.py
index bce2c0e23..65d0de316 100644
--- a/lib/meta.py
+++ b/lib/meta.py
@@ -264,7 +264,7 @@ def _get_versions(gitprefix="gitexport"):
         # If release information from release.sh exists, use that
         relinfo = {}
         with open("release_info", "rb") as relinfo_fp:
-            exec(relinfo_fp, relinfo)
+            exec(relinfo_fp.read(), relinfo)
         base_version = relinfo.get(
             "MYPAINT_VERSION_BASE",
             base_version,
